qiime/core/testing/plugin.py

```diff
@@ -38,6 +41,14 @@
 dummy_plugin.register_semantic_type(Mapping)
 dummy_plugin.register_semantic_type(FourInts)
 
+# Register formats
+dummy_plugin.register_format(IntSequenceFormat)
```

Comment:
Can both `register_format` and `register_semantic_type` be combined into plurals?

```python
plugin.register_semantic_type(FeatureTable)
plugin.register_semantic_type(Frequency)
plugin.register_semantic_type(RelativeFrequency)
plugin.register_semantic_type(PresenceAbsence)
```

to

```python
plugin.register_semantic_types(FeatureTable, Frequency, RelativeFrequency,
                               PresenceAbsence)
```